metadata.json

```diff
@@ -13,7 +13,7 @@
       "version_requirement": ">= 4.13.0 < 10.0.0"
     },
     {
-      "name": "camptocamp/augeas",
+      "name": "puppetlabs/augeas_core",
```

Comment:
the core modules are bundled into Puppet. becauss if that we dont list them in metadata.json. please remove it.